__snapshots__/packages/language-server/test-out/util/toLS.spec.js

```diff
@@ -0,0 +1,86 @@
+exports['toLS.completionItem() Should map correctly when cursor is in first line 1'] = {
+  "label": "advancement",
+  "textEdit": {
+    "newText": "advancement",
+    "insert": {
+      "start": {
+        "line": 0,
+        "character": 0
+      },
+      "end": {
+        "line": 0,
+        "character": 1
+      }
+    },
+    "replace": {
+      "start": {
+        "line": 0,
+        "character": 0
+      },
+      "end": {
+        "line": 2,
+        "character": 2
+      }
+    }
+  },
+  "insertTextFormat": 2,
+  "insertTextMode": 2
+}
+
+exports['toLS.completionItem() Should map correctly when cursor is in second line 1'] = {
+  "label": "advancement",
+  "textEdit": {
+    "newText": "advancement",
+    "insert": {
+      "start": {
+        "line": 0,
+        "character": 0
+      },
+      "end": {
+        "line": 1,
+        "character": 1
+      }
+    },
+    "replace": {
+      "start": {
+        "line": 0,
+        "character": 0
+      },
+      "end": {
+        "line": 2,
+        "character": 2
+      }
+    }
+  },
+  "insertTextFormat": 2,
+  "insertTextMode": 2
+}
+
+exports['toLS.completionItem() Should map correctly when cursor is in third line 1'] = {
+  "label": "advancement",
+  "textEdit": {
+    "newText": "advancement",
+    "insert": {
+      "start": {
+        "line": 0,
+        "character": 0
+      },
+      "end": {
+        "line": 2,
+        "character": 1
+      }
+    },
+    "replace": {
+      "start": {
+        "line": 0,
+        "character": 0
+      },
+      "end": {
+        "line": 2,
+        "character": 2
+      }
+    }
+  },
+  "insertTextFormat": 2,
+  "insertTextMode": 2
+}
```

Comment:
looks like you need to `npm run clean` (or `clean:full`) maybe? guessing these snapshots shouldnt be showing up here

Reply:
i dont know why i felt the need to highlight the entire snapshot, sorry

Reply:
i even double checked to make sure i didn't include them :sob: